dockstore-common/src/main/java/io/dockstore/common/PipHelper.java

```diff
@@ -27,7 +27,7 @@ public static String convertSemVerToAvailableVersion(String semVerString) {
             semVerString = "9001.9001.9001";
         }
         Version semVer = Version.valueOf(semVerString);
-        if (semVer.greaterThan(Version.valueOf("1.7.0"))) {
+        if (semVer.greaterThan(Version.valueOf("1.9.0"))) {
```

Comment:
Pattern has been like this and we don't actually have a 1.9.1 CLI release, but wouldn't `1.9.1` meet this condition and return `"1.10.0"`?

Reply:
In the past, we've actually used a different cwltool version in a hotfix. There's no guarantee